### PR TITLE
Disable GitHub auto-backups

### PR DESCRIPTION
**Pull request by ILIYANGERMANOV**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [ ] The PR is submitted to the `main` branch.
- [ ] I've read the **[Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md)**.
- [ ] The code builds and is tested on a real Android device.
- [ ] I confirm that I've run the code locally and everything works as expected.

## What's changed?
Describe with a few bullets **what's new:**
- a
- b
- c
- d

_💡 Tip: Please, attach screenshots and screen recordings. It helps a lot!_

## Risk Factors

**What may go wrong if we merge your PR?**

- a
- b
- c

**In what cases your code won't work?**

- a
- b
- c

## Does this PR closes any GitHub Issues?

Check **[Ivy Wallet Issues](https://github.com/Ivy-Apps/ivy-wallet/issues)**.

- Closes #{ISSUE_NUMBER}

_Replace `{ISSUE_NUMBER}` with the id/number of the issue that you've fixed._

### CODE REVIEW

**Comment by lukaskretzschmer** [comment]
Sorry, why was this disabled?

**Comment by ILIYANGERMANOV** [comment]
> Sorry, why was this disabled?

Hello, you can learn more about this in our community:
https://t.me/c/1647280565/8173/17625

TL;DR;
- Potential legal issues as I'm not sure if it's okay to use GitHub for non-dev backup purposes. So better be safe than sorry. Also, our users seems to prefer Google Drive backups which is planned
